packages/perseus/src/widgets/__tests__/radio/base-radio_test.jsx

```diff
@@ -82,17 +81,51 @@ describe("base-radio", () => {
         expect(screen.getAllByText("None of the above")).toHaveLength(2);
     });
 
+    it("registers as checked when none of the above choice is clicked", () => {
+        // Arrange
+        let updatedValues = null;
+        const onChangeHandler = (newValues) => {
+            updatedValues = newValues;
+        };
+
+        renderBaseRadio({
+            choices: [
+                generateChoice({content: "Option 1", correct: false}),
+                generateChoice({content: "Option B", correct: false}),
+                generateChoice({content: "Option Gamma", correct: true}),
+                generateChoice({content: "Option Delta", correct: false}),
+                generateChoice({
+                    isNoneOfTheAbove: true,
+                    correct: true,
+                }),
+            ],
+            onChange: onChangeHandler,
+        });
+
+        const noneOption = screen.getByRole("radio", {
+            name: "(Choice E) None of the above",
+        });
+
+        // Act
+        userEvent.click(noneOption);
+
+        // Assert
+        expect(updatedValues).toMatchObject({
+            checked: [false, false, false, false, true],
+        });
+    });
+
     describe("edit mode", () => {
         it("should render <li>'s for each choice", () => {
             // Arrange / Act
             renderBaseRadio({
-                editMode: true,
                 choices: [
                     generateChoice({content: "Option 1", correct: false}),
                     generateChoice({content: "Option B", correct: false}),
                     generateChoice({content: "Option Gamma", correct: true}),
                     generateChoice({content: "Option Delta", correct: false}),
                 ],
+                editMode: true,
```

Comment:
This whole describe is labeled as "edit mode", so I figured the tests were meant to be conducted in edit mode. I think I removed them when we were discussing removing the other editMode options. I'll go back and make this match the code on the left to remove this diff from the PR since it's just staying the way it was.